sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/aggregator/EmptyMetricData.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.internal.aggregator;
+
+import io.opentelemetry.sdk.common.InstrumentationLibraryInfo;
+import io.opentelemetry.sdk.metrics.data.Data;
+import io.opentelemetry.sdk.metrics.data.MetricData;
+import io.opentelemetry.sdk.metrics.data.MetricDataType;
+import io.opentelemetry.sdk.metrics.view.Aggregation;
+import io.opentelemetry.sdk.resources.Resource;
+
+/**
+ * This class is internal and is hence not for public use. Its APIs are unstable and can change at
+ * any time.
+ */
+public final class EmptyMetricData implements MetricData {
+
+  private static final EmptyMetricData INSTANCE = new EmptyMetricData();
+
+  private EmptyMetricData() {}
+
+  /**
+   * Get an empty instance of metric data. Empty metric data should not be used except as an
+   * indication that recordings were dropped via {@link Aggregation#drop()}.
+   *
+   * @return an empty MetricData.
+   */
+  public static MetricData getInstance() {
+    return INSTANCE;
+  }
+
+  @Override
+  public Resource getResource() {
+    throw new UnsupportedOperationException("EmptyMetricData does not support getResource().");
```

Comment:
Perhaps there isn't much point but let's go ahead and add unit tests for this class specifically that just check `.isInstanceOf` on the exception. It can be considered a double-check that the semantics are indeed what we intended